js/project.js

```diff
@@ -0,0 +1,121 @@
+const worksTargetElement = document.querySelector('#works-target-element');
+
+tags = ['CANOPY', 'Back End Dev', '2015'];
+links = ['html', 'Ruby on rails', 'css', 'javascript'];
+
+const projects = [
+  {
+    id: 'tonic-element',
+    title: 'Tonic',
+    tags: ['CANOPY', 'Back End Dev', '2015'],
+    screenshot: 'images/snapshoot-portfolio-profile-img.png',
+    screenshotalt: 'Screenshot of Tonic Project',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    links: ['html', 'Ruby on rails', 'css', 'javascript'],
+    btnId: 'tonic-btn-element',
+  },
+  {
+    id: 'availability-element',
+    title: 'Multi-Post Stories',
+    tags: ['CANOPY', 'Back End Dev', '2015'],
+    screenshot: 'images/snapshoot-portfolio-availability.png',
+    screenshotalt: 'Screenshot of Multi-Post Stories Project',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    links: ['html', 'Ruby on rails', 'css', 'javascript'],
+    btnId: 'availability-btn-element',
+  },
+  {
+    id: 'weather-element',
+    title: 'Facebook 360',
+    tags: ['CANOPY', 'Back End Dev', '2015'],
+    screenshot: 'images/snapshoot-portfolio-weather.png',
+    screenshotalt: 'Screenshot of Facebook 360 Project',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    links: ['html', 'Ruby on rails', 'css', 'javascript'],
+    btnId: 'weather-btn-element',
+  },
+  {
+    id: 'art-element',
+    title: 'Uber Navigation',
+    tags: ['CANOPY', 'Back End Dev', '2015'],
+    screenshot: 'images/snapshoot-portfolio-art.png',
+    screenshotalt: 'Screenshot of Uber Navigation Project',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    links: ['html', 'Ruby on rails', 'css', 'javascript'],
+    btnId: 'art-btn-element',
+  },
+  {
+    title: 'Window Pop-up',
+    tags: ['CANOPY', 'Back End Dev', '2015'],
+    screenshot: 'images/snapshoot-portfolio-availability.png',
+    screenshotalt: 'Screenshot of Window pop-up Project',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    links: ['html', 'Ruby on rails', 'css', 'javascript'],
+    btnId: 'window-btn-element',
+  },
+  {
+    title: 'Art Work',
+    tags: ['CANOPY', 'Back End Dev', '2015'],
+    screenshot: 'images/snapshoot-portfolio-weather.png',
+    screenshotalt: 'Screenshot of Art Work Project',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
```

Comment:
Please change the content of this array objects, they should not be the same thing, each object should have unique values